Tests/SentryTests/Integrations/Feedback/SentryFeedbackTests.swift

```diff
@@ -29,15 +29,48 @@ class SentryFeedbackTests: XCTestCase {
     }
     
     func testSerializeWithAllFields() throws {
-        let sut = SentryFeedback(message: "Test feedback message", name: "Test feedback provider", email: "test-feedback-provider@sentry.io", screenshot: Data())
+        let sut = SentryFeedback(message: "Test feedback message", name: "Test feedback provider", email: "test-feedback-provider@sentry.io", attachments: [Data()])
         
         let serialization = sut.serialize()
         XCTAssertEqual(try XCTUnwrap(serialization["message"] as? String), "Test feedback message")
         XCTAssertEqual(try XCTUnwrap(serialization["name"] as? String), "Test feedback provider")
         XCTAssertEqual(try XCTUnwrap(serialization["contact_email"] as? String), "test-feedback-provider@sentry.io")
         XCTAssertEqual(try XCTUnwrap(serialization["source"] as? String), "widget")
         
-        let attachments = sut.attachments()
+        let attachments = sut.attachmentsForEnvelope()
+        XCTAssertEqual(attachments.count, 1)
+        XCTAssertEqual(try XCTUnwrap(attachments.first).filename, "screenshot.png")
+        XCTAssertEqual(try XCTUnwrap(attachments.first).contentType, "application/png")
+    }
+    
+    func testSerializeCustomFeedback() throws {
+        let sut = SentryFeedback(message: "Test feedback message", name: "Test feedback provider", email: "test-feedback-provider@sentry.io", source: .custom, attachments: [Data()])
+        
+        let serialization = sut.serialize()
+        XCTAssertEqual(try XCTUnwrap(serialization["message"] as? String), "Test feedback message")
```

Comment:
`l`: see my comment in #4784

https://github.com/getsentry/sentry-cocoa/pull/4874#discussion_r1965185356

Reply:
Responded!